scrapers/ar/__init__.py

```diff
@@ -196,6 +196,15 @@ class Arkansas(State):
             "name": "2022 Fiscal Session",
             "start_date": "2022-02-14",
             "end_date": "2022-03-15",
+            "active": False,
+        },
+        {
+            "_scraped_name": "Third Extraordinary Session, 2022",
+            "classification": "special",
+            "identifier": "2021S3",
```

Comment:
Need to swap this to 2022S3, but bills posted now

Reply:
I left it at 2021 because it's the third special of the 2021 biennium, and that's the coding we've use for the other specials in that state.

Reply:
Oh I see the issue now, that won't scrape correctly =P. I'll change that and merge in a bit.